README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
+| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | C                                                                |
 | Unicode-free output                                                                       | 🚧      | ✖️                                                           | ✔️                                                                |
-| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | ?                                                                |
-| Coloured diffing                                                                          | ✔️       | ✔️                                                           | ✖️                                                                |
+| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
+| Coloured diffing                                                                          | ✔️       | ✔️                                                           | C                                                                |
 | multi-line diffing                                                                        | ✔️       | ✖️                                                           | ✖️                                                                |
 | Contextual failures                                                                       | ✔️       | ✖️                                                           | ✖️                                                                |
-| Inter-test progress output during test suite execution                                    | ✔️       | ✔️                                                           | ?                                                                |
-| Intra-test progress output during test suite execution                                    | 🚧      | ✔️                                                           | ?                                                                |
+| Inter-test progress output during test suite execution                                    | ✔️       | ✔️                                                           | ✔️                                                                |
+| Intra-test progress output during test suite execution                                    | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-quickcheck)      |
```

Comment:
Intray-test output is where you see the number of property tests that have already passed go up during execution.